python/py-makelive/Portfile

```diff
@@ -32,3 +32,7 @@ checksums           rmd160  efe2a8db5b43e6ba40f4988e465f4f0911755acd \
                     size    14016
 
 python.versions     312
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-flit
```

Comment:
instead of doing this, which of course works, there are a few pre-defined PEP517 backends set in the `python` PortGroup. So you could just do (without the need for the `if`-statement): ```python.pep517_backend flit```

Reply:
Ok thanks, I had never heard of a 'PEP517 backend' until just now. I don't work with Python very often.